system/churchcore/classes/qqFileUploader.class.php

```diff
@@ -12,7 +12,7 @@ class qqFileUploader {
    * create qqUploadedFileXhr or qqUploadedFileForm depending of variable set (get or file)
    *
    * @param array $allowedExtensions
-   * @param number $sizeLimit
+   * @param int $sizeLimit
    */
   function __construct(array $allowedExtensions = array(), $sizeLimit = 10485760) {
```

Comment:
Can we use size limit as a constant, instead of repeating it all over again? Although not your code. But you're touching that part already anyway. :wink: